src/resources/views/crud/fields/radio.blade.php

```diff
@@ -1,6 +1,7 @@
 <!-- radio -->
 @php
-    $optionValue = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? '';
+    $optionValue = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? 0;
+    $optionValue = is_bool($optionValue) ? (int)$optionValue : $optionValue;
```

Comment:
Can you please explain why this is needed? I'm afraid I don't understand. Could you not just pass booleans when you define the field?